literature.bib

```diff
@@ -4047,6 +4158,40 @@ @article{MJ:TAOSD14
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2014 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{LMS:SEFM14,
+	title={{A Thread-Safe Library for Binary Decision Diagrams}},
+	author={Lovato, Alberto and Macedonio, Damiano and Spoto, Fausto},
+	booktitle=SEFM,
+	pages={35--49},
+	publisher = Springer,
+	address = BerlinHeidelberg,
+	year = 2014
+}
+
+@article{BCHY:Heuristics14,
+	title={{BDD-Based Heuristics for Binary Optimization}},
+	author={Bergman, David and Cir{\'{e}}, Andr{\'{e}} A and van Hoeve, Willem-Jan and Yunes, Tallys},
+	journal={J.\ of Heuristics},
+	volume={20},
+	number={2},
+	pages={211--234},
+	publisher = Springer,	
+	address = BerlinHeidelberg,
+	month=APR,
+	year=2014,
+}
+
+@article{BCHH:IJC14,
+	title={{Optimization Bounds from Binary Decision Diagrams}},
+	author={Bergman, David and Cire, Andre A and van Hoeve, Willem-Jan and Hooker, John N},
+	journal={INFORMS J.\ on Computing},
```

Comment:
Why no abbreviation for this journal? Is in here several times.

Reply:
Added.